doc/user_guide/libraries.rst

```diff
@@ -108,17 +108,18 @@ the ``lib/read_only`` `directory
 For detailed instructions on how to build and use these libraries
 please refer to their specific ``README.md`` documentation.
 
-NAN Test
-^^^^^^^^
+Value Range Check
+^^^^^^^^^^^^^^^^^
 
-These libraries test all input and output parameters of a kernel to
-make sure they are not ``NaN`` or infinite.  More information can be
-found in the :ref:`NAN Test <psydata_nan_test>` section.
+These libraries can test if user-defined variables are within a
+specified range. Additionally, they also verify that they are
+not ``NaN`` or infinite.  More information can be
```

Comment:
Fair enough. It is probably even required, since otherwise we would have to look how conditionals with NANs are handled (always false, but it depends on how the if-statements are written - in this case I believe a NAN might actually go undetected:
```
            {{indent}}if (value < this%value_ranges(this%next_var_index,1)  .or.               &
            {{indent}}    value > this%value_ranges(this%next_var_index,2)) then
            {{indent}}    write(stderr, '(13G0)') "PSyData: Variable '",                       &
```
If a comparison with NAN is always False, no warning would be printed. So good suggestion to leave it in (but de-facto I was thinking about this when writing the statements, and I originally had them reversed: `if (...) then !nothing else print warning ` - but that looks horrible :)  )